src/App.tsx

```
@@ -59,9 +59,28 @@ function nextStep(
-
+  const [typedWord, setTypedWord] = useState("")
   const { createEnemies, getEnemies, setEnemies } = useEnemyStore();
-  const { getWords } = useWordStore();
+
+  const onWordTyped = (e: ChangeEvent<HTMLInputElement>) => {
+    setTypedWord(e.target.value);
+    const enemies = getEnemies();
+    const updated_enemy = enemies.find(({focus, status}: {focus: boolean, status: Status}) => {
+        return focus == true && status == Status.Active
+    })
+    let new_enemies = enemies!.map((enemy: EnemyObject) => {
```

ESLint

[error] 'new_enemies' is never reassigned. Use 'const' instead.

(prefer-const)